example/ck_tile/01_fmha/fmha_fwd.cpp

```diff
@@ -149,9 +169,32 @@ bool run(const ck_tile::ArgParser& arg_parser)
     if(scale == .0f)
         scale = 1.0 / ck_tile::sqrt(static_cast<float>(hdim_q)); // TODO: q ? v ?
 
-    float descale_q = arg_parser.get_float("descale_q");
-    float descale_k = arg_parser.get_float("descale_k");
-    float descale_v = arg_parser.get_float("descale_v");
+    bool squant = arg_parser.get_bool("squant");
+    if constexpr(!std::is_same_v<DataType, ck_tile::fp8_t>)
+    {
+        if(squant)
+            std::cerr << "static quantization on support fp8 for now" << std::endl;
+        return false;
+    }
+
+    float range_q = arg_parser.get_float("range_q");
+    float range_k = arg_parser.get_float("range_k");
+    float range_v = arg_parser.get_float("range_v");
+    float range_p = arg_parser.get_float("range_p");
+    float range_o = arg_parser.get_float("range_o");
+
+    float max_dtype = ck_tile::type_convert<float>(ck_tile::numeric<DataType>::max());
+
+    float scale_p = 1.f;
+    float scale_o = 1.f;
+
+    if(squant)
+    {
+        scale   = scale * (range_q / max_dtype) * (range_k / max_dtype);
+        scale_p = max_dtype / range_p;
+        // scale_p = [max(fp8_t)/range_o] * [range_p/max(fp8_t)] * [range_v/max(fp8_t)]
+        scale_o = range_p * range_v / range_o / max_dtype;
```

Comment:
suggest to rename to `dtype_max`, since the max is a noun not a adjective

Reply:
c9b59e398b76528a73a784ae91adc19b5fd9d72d